lib/treatments.js

```diff
@@ -88,9 +123,9 @@ function storage (env, ctx) {
 
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [This function's cyclomatic complexity is too high. (8)](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2042010566/issues/source?bid=2452374&fileBranchId=3130808#l124)